src/ray/rpc/server_call.h

```diff
@@ -141,6 +139,8 @@ class ServerCallImpl : public ServerCall {
     // NOTE(hchen): This `factory` local variable is needed. Because `SendReply` runs in
     // a different thread, and will cause `this` to be deleted.
     const auto &factory = factory_;
+    // Create a new `ServerCall` to accept the next incoming request.
+    factory.CreateCall();
```

Comment:
Can you add some more comments here on the reasoning for putting it here.

Reply:
Yes good idea